conf/application.conf

```diff
@@ -84,6 +84,19 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "config.DIModule"
 
+# Logger
```

Comment:
Is there a reason why this was added back in? This was removed during the dependency upgrade using  the information gotten from [here](https://confluence.tools.tax.service.gov.uk/display/CD/Log+file+configuration+in+MDTP) and it has deprecated values which is highlighted [here](https://github.com/playframework/playframework/blob/master/core/play-guice/src/main/scala/play/api/inject/guice/GuiceApplicationBuilder.scala#L204) and [here](https://github.com/playframework/playframework/blob/master/core/play-guice/src/main/scala/play/api/inject/guice/GuiceApplicationBuilder.scala#L134)